src/metadata/selectors.js

```diff
@@ -74,8 +74,10 @@ export const getCategoryOptionsByCategoryOptionComboId = createCachedSelector(
         const categoryCombos = Object.values(metadata.categoryCombos)
 
         for (const categoryCombo of categoryCombos) {
-            for (const curCategoryOptionCombo of categoryCombo.categoryOptionCombos) {
-                if (curCategoryOptionCombo.id === categoryOptionComboId) {
+            const curCategoryOptionCombos =
+                categoryCombo?.categoryOptionCombos || []
```

Comment:
How come this `?.` is necessary? I would not expect `categoryCombos` to contain a null or undefined value? 
It's a minor thing, I'm just curious.

Reply:
I don't remember to be honest.. I just encountered this once, but that might be due to my trying things out or due to some hot reloading weirdness